website/src/components/Tooltip/key-terms.json

```diff
@@ -824,7 +824,7 @@
     "link": "/docs/tech-explanation/glossary#orphan-rate-target"
   },
   "orphan transaction": {
-    "content": "Orphan transactions are those whose parental transactions are missing at the time that they are processed.",
+    "content": "An orphan transaction is one for which the parent transaction is absent at the time of its processing.",
```

Comment:
Instead of directly changing the description in this file, you might want to modify the first sentence of "orphan transaction" in the glossary.md. Then you can run `yarn gen-term` to update this file.